src/app/shared/components/header/header.component.ts

```diff
@@ -64,6 +53,26 @@ export class PLHMainHeaderComponent implements OnInit, OnDestroy {
       });
     }
   }
+
+  subscribeToAppConfigChanges() {
+    this.appConfigChanges$ = this.appConfigService.appConfig$.subscribe((appConfig: IAppConfig) => {
+      this.headerConfig = appConfig.APP_HEADER_DEFAULTS;
+      if (this.headerConfig.colour === "none") {
+        this.colour = undefined;
+      } else if (COLOUR_OPTIONS.includes(this.headerConfig.colour)) {
+        this.colour = this.headerConfig.colour;
+      } else {
+        this.colour = "primary";
+      }
```

Comment:
nit(non-blocking): might be a bit cleaner defining the options as a map/enum so that we can just pick `COLOUR_OPTIONS[this.headerConfig.colour] || 'primary'` instead of all the if statements